### PR TITLE
Additional code for V9>v10 module migration

### PR DESCRIPTION
**Pull request by pwatson100**

### CODE REVIEW

File: css/symbaroum-dnd5e-mod.css [hunk 1]
```diff
@@ -632,13 +633,17 @@ a.inline-roll {
   margin: 0px 20px 0px 10px;
 }
 
-.symbaroum-dnd5e-mod h1 {
-  /* font-family: 'Jim Nightshade'; */
+/* .symbaroum-dnd5e-mod h1 {
+  font-family: 'Primitive';
+  font-size: 68pt;
+} */
```
Comment (trioderegion) [inline css/symbaroum-dnd5e-mod.css:639]: Remove commented code (git remembers all)

File: css/symbaroum-dnd5e-mod.css [hunk 2]
```diff
@@ -648,25 +653,49 @@ a.inline-roll {
   font-family: 'Primitive';
   font-size: 25pt;
 }
-.symbaroum-dnd5e-mod h3 {
+/* .symbaroum-dnd5e-mod h3 {
   padding: 0;
   margin: 0;
   margin-top: 5px;
   font-size: 14pt;
+} */
```
Comment (trioderegion) [inline css/symbaroum-dnd5e-mod.css:661]: Remove commented code (git remembers all)

File: css/symbaroum-dnd5e-mod.css [hunk 3]
```diff
@@ -648,25 +653,49 @@ a.inline-roll {
...
+
+/* .symbaroum-dnd5e-mod h5 {
   font-weight: bold;
   font-family: 'Fondamento';
   font-size: 12pt;
   color: hsl(19, 70%, 37%);
   display: block;
   margin: 0px 20px 0px 10px;
-}
+} */
```
Comment (trioderegion) [inline css/symbaroum-dnd5e-mod.css:698]: Remove commented code (git remembers all)

File: scripts/core.js [hunk 4]
```diff
@@ -64,11 +66,13 @@ Hooks.on('init', () => {
...
   Initialize all Sub Modules
   */
-  Object.values(SUB_MODULES).forEach( (cl) => {
-    logger.info(COMMON.localize('SYB5E.Init.SubModule', {name: cl.NAME}));
+  Object.values(SUB_MODULES).forEach((cl) => {
+    logger.info(COMMON.localize('SYB5E.Init.SubModule', { name: cl.NAME }));
     cl.register();
   });
 
+  Journal.registerSheet('SYB5E', SymbaroumWide, { label: game.i18n.localize('SYB5E.journal.widejournal.name'), makeDefault: false });
```
Comment (trioderegion) [inline scripts/core.js:74]: Move this registration to a 'register' function within 'SymbaroumWide', similar to the other registration functions.

File: scripts/modules/journal.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+//             }
+
+//             static defaults() {
+//                 SymbaroumJournalSheet.defaults(this);
+//             }
+//         }
+//         SymbaroumWide.register();
+
+//     }
+// }
```
Comment (trioderegion) [inline scripts/modules/journal.js:103]: Remove commented code (git remembers all)

**Review by trioderegion — CHANGES_REQUESTED** [review]
Mostly just cleanup changes. One actual code change.

File: scripts/core.js (hunk 4 above)
Comment (trioderegion) [reply scripts/core.js]: Resolved in update.

**Review by trioderegion — CHANGES_REQUESTED** [review]
Just the css commented lines remain.

File: css/symbaroum-dnd5e-mod.css (hunk 1 above)
Comment (pwatson100) [reply css/symbaroum-dnd5e-mod.css]: Removed

**Review by trioderegion — APPROVED** [review]

**Pull request merged**